test/lib/prisma.test.ts

```diff
@@ -0,0 +1,101 @@
+
+import {it, expect} from '@jest/globals';
+import {whereAndSelect} from "@lib/prisma";
+
+it('can decorate using whereAndSelect', async () => {
```

Comment:
Codacy has a fix for the issue: [Replace `'can·decorate·using·whereAndSelect'` with `"can·decorate·using·whereAndSelect"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7713215)
```suggestion
it("can decorate using whereAndSelect", async () => {
```